src/Components/allWebsite.jsx

```diff
@@ -0,0 +1,27 @@
+import React,{useState,useEffect} from 'react';
+import {GetWebsitesApi} from '../Api/api'
+
+
+export const WebsiteList = () => {
+  const [websites, setWebsites] = useState([]);
+
+	useEffect(() => {
+		const fetchData = async () => {
+			const data = await GetWebsitesApi();
+      console.log(data);
+			setWebsites(data);
+		};
+		fetchData();
+	}, []);
+  
+  return <>
+  {
+    websites.map((w)=><>
```

Comment:
add key to list item

Reply:
מזה אומר key? 
גם ב-consol הוא עושה לי על זה הערה